easybuild/easyconfigs/p/pubtcrs/pubtcrs-f30b96a-intel-2019.01.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'MakeCp'
+
+name = 'pubtcrs'
+version = 'f30b96a'
+
+homepage = 'https://github.com/phbradley/pubtcrs'
+description = """This repository contains C++ source code for the TCR clustering 
+and correlation analyses described in the manuscript "Human T cell receptor 
+occurrence patterns encode immune history, genetic background, and receptor 
+specificity" by William S DeWitt III, Anajane Smith, Gary Schoch, John A Hansen, 
+Frederick A Matsen IV and Philip Bradley, available on bioRxiv."""
+
+toolchain = {'name': 'intel', 'version': '2019.01'}
+
+source_urls = ['https://github.com/phbradley/pubtcrs/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['077707390aef4cb617f3b9a0d077a3511205530ac523dbb486af5c832379226e']
+
+dependencies = [
+    ('Boost', '1.69.0'),
+]
+
+
```

Comment:
@sassy-crick style nitpicking: please drop duplicate empty line

Reply:
@boegel I have updated the BOOST version to 1.70.0 and also updated the build-chain to intel-2019a.